NuGet.config

```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
     <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="DotNet-Core" value="https://myget.org/f/dotnet-core/api/v3/index.json" />
```

Comment:
We should mirror them once things stabilize. Now there's a lot of churn and we usually need new packages immediately